src/SamplesServiceModelGenerator/Swagger/Parser.cs

```diff
@@ -439,5 +458,15 @@ private void AdjustResponse(OperationResponse response, string jsonText)
             ParseRef(response.Schema, schema);
             ParseRef(response.Schema.Items, schema.Object("items"));
         }
+
+        private static List<TResult> MapJsonText<T, TResult>(string jsonText, Func<string, string, TResult> selector)
```

Comment:
Could you explain what the parsing issue was, and how this PR fixes it?

Right now, without any tests, I don't really understand what this is achieving, or how it's parsing differently.
The old version already had access to a `JsonObject`, from which you could access properties by key if it exists. It seems like this approach is re-serializing the `JsonObject` to a string, calling `MapJsonText<...>()` to re-deserialize into a `JsonObject` or `JsonDocument`, and then applying the same selector as before. This seems like a redundant serialization/deserialization

Reply:
Have you tested the .net sdk locally with updated Samples service models?